Nos/Controller/SearchController.swift

```diff
@@ -101,8 +146,28 @@ class SearchController: ObservableObject {
         }
     }
     
-    func clear() {
-        query = ""
-        authorSuggestions = []
```

Comment:
Don't worry -- this isn't deleted -- I just moved it up there. ☝️☝️ In a previous life I got into the habit of alphabetizing almost everything, and I might like to continue that habit since it makes things _much_ easier for me to find. And as a bonus, it takes away any need for thinking about _where_ a new method or property or import should go.